upgrade/UPGRADE-v9.1.2-dev.md

```diff
@@ -63,3 +63,6 @@ There you can find links to upgrade notes for other versions too.
 
 - update your data fixtures to always generate data with same prices ([#2356](https://github.com/shopsys/shopsys/pull/2356))
     - see #project-base-diff to update your project
+
+- increase minimal version of `symfony/proxy-manager-bridge` package ([#2359](https://github.com/shopsys/shopsys/pull/2359))
+    - see #project-base-diff to update your project
```

Comment:
You should run `composer update symfony/proxy-manager-bridge` too 😉

Reply:
added info, thanks 👍